### PR TITLE
Change logo from SVG to text for better compatibility between browsers.

### PR DESCRIPTION
**Pull request by BrOrlandi**
image

### CODE REVIEW

**Comment by BrOrlandi** [comment]
@BrOrlandi Teste

File: src/styles/animation.styl [hunk 1]
```diff
@@ -46,8 +46,16 @@
...
-      font-size: 210px;
+    .logoText {
+      color: black;
+      text-shadow: stroke(5, $titlesColor);
+      letter-spacing: .18em;
+      font-size: 200px;
+      line-height: 1em;
+
+      div {
+        white-space: nowrap;
```
Comment (luanorlandi) [inline src/styles/animation.styl:60]: Check this for long lines:

```suggestion
        white-space: nowrap;
        margin-left: -100%;
        margin-right: -100%;
```

image

**Review by luanorlandi — CHANGES_REQUESTED** [review]

File: src/styles/animation.styl (hunk 1 above)
Comment (BrOrlandi) [reply src/styles/animation.styl:60]: Done: f4eb8aca9f5dc4ccf1e9eaa3a079efb2c7ae235a

Comment (luanorlandi) [reply src/styles/animation.styl:60]: 👌

**Review by luanorlandi — APPROVED** [review]

**Pull request merged**